mkt/developers/utils.py

```diff
@@ -154,3 +154,14 @@ def escalate_prerelease_permissions(app, validation, version):
         escalate_app(
             app, version, nobody, 'App uses prerelease permissions',
             mkt.LOG.ESCALATION_PRERELEASE_APP)
+
+
+def prioritize_app(app, user):
+    if not app.priority_review:
```

Comment:
I guess you could split this function in two as well. Have a `notify_app_prioritization` function and then this can be:

```py
def prioritize_app(app, user):
    if not app.priority_review:
        app.update(priority_review=True)
        notify_app_prioritization(app, user)
```

But then we're writing a function that only gets called once, so that might not be better.